docs/advanced/exceptions.rst

```diff
@@ -328,6 +328,28 @@ Alternately, to ignore the error, call `PyErr_Clear
 Any Python error must be thrown or cleared, or Python/pybind11 will be left in
 an invalid state.
 
+Handling warnings from the Python C API
+=====================================
```

Comment:
Could you please add a couple more `=`, to line up with the text above?

(I overlooked this before.)